extensions/ccui/uiwidgets/UIButton.js

```diff
@@ -221,9 +217,12 @@ ccui.Button = ccui.Widget.extend(/** @lends ccui.Button# */{
      * @returns {cc.Size}
      */
     getVirtualRendererSize: function(){
+        if (this._unifySize)
+            return this.getNormalSize();
+
         var titleSize = this._titleRenderer.getContentSize();
         if (!this._normalTextureLoaded && this._titleRenderer.getString().length > 0) {
-            return titleSize;
+            return cc.size(titleSize);
```

Comment:
All `getContentSize` in Cocos2d-h5 return a new `cc.Size` object, so it needn't create a new size again.